lib/diagnostic.md

```diff
@@ -12,7 +18,10 @@ Using the diagram in the `README` file (also in the `assets` folder), what are t
 you could return ALL list items (`li`) using jQuery? This is intentionally vague. (Assume starting at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+```
+console.log( $( "li" ).get() );
+$( "ul" ).find( "li" )
+```
```

Comment:
You don't need to console log when using jQuery in the browser.  Try typing something similar to this in the browser (without console.log) and check what it returns.

Correct answer: $('.li') or $('ol').children()